### PR TITLE
Add HasEitherUnicodeFlag abstract operation

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

**Review by michaelficarra — APPROVED** [review]
LGTM

File: spec.html [hunk 1]
```diff
@@ -36748,6 +36743,20 @@ <h1>
...
+              _rer_: a RegExp Record,
+            ): a Boolean
+          </h1>
+          <dl class="header">
+          </dl>
+          <emu-alg>
+            1. If _rer_.[[Unicode]] is *true* or _rer_.[[UnicodeSets]] is *true*, then
+              1. Return *true*.
+            1. Return *false*.
+          </emu-alg>
```
Comment (michaelficarra) [inline spec.html:36758]: You forgot to end your `<emu-clause>` here.

Comment (ljharb) [reply spec.html:36758]:
```suggestion
          </emu-alg>
        </emu-clause>
```

Comment (mathiasbynens) [reply spec.html:36758]: Cheers!

**Pull request merged**